server/unittools.cpp

```diff
@@ -1319,13 +1320,24 @@ bool bounce_path_constraint::is_allowed(
 /**
  * Move or remove a unit due to stack conflicts. This function will try to
  * find a random safe tile within a given distance of the unit's current tile
- * and move the unit there. If no tiles are found, the unit is disbanded. If
- * 'verbose' is true, a message is sent to the unit owner regarding what
- * happened (the exact message depends on 'reason'). The maximum distance it
- * 2 by default for backward compatibility.
+ * and move the unit there. If no tiles are found, the unit is disbanded. The
+ * maximum distance it 2 by default for backward compatibility. A function
+ * reference may be provided to handle successfully finding a bounce
+ * location. By default this will report the bounce to the unit's owner
+ * before the unit is moved. A function reference may also be provided to
+ * handle failing to find a bounce location. By default this will report the
+ * failed bounce to the unit's owner before the unit is disbanded.
+ *
+ * See `bounce_unit_silently` to attempt a bounce without reporting it to the
+ * unit's owner.
+ *
+ * Note: The unit may have died even if the bounce was successful.
+ * Note: If a transport fails to bounce, all units in the transport will be
+ * bounced, triggering on_success or on_fail for each.
```

Comment:
Note: Doxygen has a special command for this, called [`\note`](https://www.doxygen.nl/manual/commands.html#cmdnote) (or `@note` in JavaDoc style). It renders a bit nicer (if one can call the Doxygen output *nice*).